src/components/AssocTable.vue

```diff
@@ -585,6 +598,9 @@ export default {
           assocSubject: subjectElem.label,
           subjectCurie: subjectElem.id,
           subjectLink,
+          title: truncate(elem.publicationMeta.title, 20),
```

Comment:
`publicationMeta` is sometimes undefined, causing this reference to `title` to throw an error.

![Screen Shot 2021-07-30 at 10 09 39 AM](https://user-images.githubusercontent.com/312923/127681173-145d4f70-cf32-4b3a-b622-cffaa0e23f76.png)

The error reliably occurs if you go to https://deploy-preview-438--monarch-ui.netlify.app/disease/MONDO:0007947, click "Publication" in the sidebar, then browse to page 4.

Reply:
Also, just an opinion, but 20 characters for a lot of these titles isn't much. I'd prefer to see more and possibly the option to expand the ellipses into the full title.